srcDoc/command_line_parameters.rst

```diff
@@ -345,61 +345,61 @@ Test Parameters for Publishing and Subscribing Applications
 
 -  ``-peer <address>|<addres>[:<id>]``
 
-   Adds a peer to the peer host address list. If ``-rawTransport`` is used, a
-   optional ID of the subscriber could beprovied. This argument may be repeated
+   Adds a peer to the peer host address list. If ``-rawTransport`` is used, 
+   you can provide an optional subscriber ID. This argument may be repeated
    to indicate multiple peers.
 
    **Default:**
    ``Not set. RTI Perftest will use the default initial peers (localhost, shared-memory and multicast).``
 
 -  ``-threadPriorities X:Y:Z``
 
-    This Command Line parameter is supported only for the C++ and C++03 API
-    Implementations.
+    This command-line parameter is supported only for the C++ and C++03 API
+    implementations.
 
-    Set the priorities for the application Threads:
-        X -- For the Main Thread, which will be the one sending the data. Also
+    Set the priorities for the application threads:
+        X -- For the Main Thread, which will be the one sending the data, or 
              for the Asynchronous thread if that one is used.
-        Y -- For the Receive Threads, If the -useReadThread is used, also for
+        Y -- For the Receive Threads or, if the -useReadThread is used, for
              the thread created to receive and process data.
         Z -- For the rest of the threads created by the middleware: Event and
              Database Threads.
 
     Three default values: h (high), n (normal) and l (low) can be used
     instead of numbers.
 
-    To see what values can be used for the different threads see
-    *RTI Connext DDS Core Libraries Platform Notes Version 5.3.1*
+    To see what values can be used for the different threads, see
+    *RTI Connext DDS Core Libraries Platform Notes Version 5.3.1*.
```

Comment:
That is a mistake, it should not specify any version.

Reply:
Change to:
see the following tables in the *RTI Connext DDS Core Libraries Platform Notes*.

Change each bullet to this format:
- the "Thread-Priority Definitions for Linux Platforms" table

Reply:
Sounds good

Reply:
done